data/ecosystem/vendors.yaml

```diff
@@ -91,7 +91,7 @@
   oss: false
   commercial: true
 - name: Grafana Labs
-  distribution: false
+  distribution: true
   nativeOTLP: true
   url: 'https://grafana.com/oss/opentelemetry/'
   contact: ''
```

Comment:
While updating this, can you please also list a contact, thanks!

Reply:
Yeah, adding @jpkrohling's github account here.

Reply:
@jpkrohling: please approve this PR so I know you are OK with that:)

Reply:
Done.